content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -455,6 +558,9 @@ journal, and click the :menuselection:`⚙️ (gear) icon --> Invalidate Number
 .. image:: brazil/range-number-invalidation-wizard.png
    :alt: Number range invalidation wizard in Odoo.
 
+.. important::
+   This electronic document is only available for NF-e.
```

Comment:
Specify which document exactly (for example, "Invalidate invoice number range documents are only available for electronic invoices for goods (NF-e)."